src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,4 +1,14 @@
-// Add the required types and props
-export const UserInfo = () => (
-  <>UserInfo markup</>
+import { User } from '../../types/User';
+import './UserInfo.scss';
+
+interface Props {
+  user: User;
+}
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
+  <>
+    <a className="UserInfo" href={`mailto:${user.email}`}>
+      {user.name}
+    </a>
+  </>
 );
```

Comment:
Remake with destructuring
```suggestion
export const UserInfo: React.FC<Props> = ({ user }) => {
	const {
		email,
		name
	} = user;

	return (
	  <>
	    <a className="UserInfo" href={`mailto:${email}`}>
	      {name}
	    </a>
	  </>
	);
};
```

Reply:
done